pkg/config/cnf.go

```diff
@@ -74,24 +74,6 @@ type Operator struct {
 	// If its all namespace then you can replace it with ALL_NAMESPACE TODO: add check for ALL_NAMESPACE
 	Namespace string `yaml:"namespace" json:"namespace"`
 
-	// Status is a required field , specified what status of the csv to be checked.
-	Status string `yaml:"status" json:"status"`
```

Comment:
So I take it that these fields were added but never written?  They must have been placeholders for https://issues.redhat.com/browse/CTONET-615.